lib/commands/WeatherCommand.js

```diff
@@ -0,0 +1,27 @@
+const weather = require('openweathermap-js')
+
+const Command = require('./Command')
+
+module.exports = new Command({
+	name: 'weather',
+	command: getWeather
+})
+
+function getWeather(context, zip) {
+	const { logger, message } = context 
+	weather.defaults({
+		appid: '6382f16f08da7243484a4041a62ce7da',
```

Comment:
We should store this in `conf/app.json` and pull it in here instead. You can access the configuration by changing line 11 above to:

```
const { config, logger, message } = context
```